package.json

```diff
@@ -40,20 +39,21 @@
     "printf": "^0.6.1",
     "set-value": "^4.0.1",
     "ssri": "^8.0.1",
-    "xmldom": "github:xmldom/xmldom#0.7.5"
+    "xmldom": "github:xmldom/xmldom#0.7.5",
+    "ember-style-modifier": "^0.8.0"
   },
   "dependencies": {
     "@glimmer/component": "^1.0.2",
     "@glimmer/tracking": "^1.0.2",
     "autosize": "^4.0.4",
-    "ember-animated": "^0.11.0",
-    "ember-auto-import": "^1.6.0",
-    "ember-cli-babel": "^7.22.1",
+    "ember-animated": "^1.0.4",
+    "ember-auto-import": "^2.4.2",
```

Comment:
I think this is what makes it a breaking change for LSB or anything still on Ember 3.x.

You have to have all dependencies using _either_ `ember-auto-import` 1.x OR 2.x -- you can't mix-and-match. 😬